docs/settingUp.md

```diff
@@ -111,6 +113,24 @@ Eclipse IDE is our preferred development environment. Support requests related t
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
+### IntelliJ IDEA
+1. Download and install [IntelliJ IDEA Ultimate Edition](https://www.jetbrains.com/idea/).
+   * You will need an Ultimate Edition license in order to work with Google App Engine in IntelliJ IDEA.
+   * Sign up for the free [JetBrains student license](https://www.jetbrains.com/student/) if you are a student registered in an educational institution.
+1. Configure IntelliJ
+   * Click `Configure → Project Defaults → Project Structure`.
+   * Under `Project SDK`, click `New → JDK`. Locate the `Java` folder where you have installed `JDK 1.7`.
+   * Select `jdk1.7.*` and click `OK`.
+   * Click `OK`.
+1. Configure Indentation: In TEAMMATES, we use 4 spaces in place of tabs for indentations.
+   * Go to `File → Settings → Editor → Code Style` and ensure that `Use tab character` is unchecked for `Java`, `JavaScript`, `HTML`, `CSS` and `XML`.
```

Comment:
Text encoding is set to UTF-8 by default but I added a step to confirm that it is indeed UTF-8.
The configuration of App Engine takes place during the setting up of the `dev server` run configuration.